src/main/java/racing/Engine.java

```diff
@@ -0,0 +1,18 @@
+package racing;
+
+import java.util.Random;
+
+public class Engine {
+    private final static Integer MAX_RANDOM_NUMBER = 10;
+    private final static Integer REFERENCE_NUMBER = 4;
```

Comment:
Java Language Specification에 따라, static 키워드를 접근 제어자 다음으로 옮겨보면 좋을 것 같아요.
https://eblo.tistory.com/12

```suggestion
    private static final Integer MAX_RANDOM_NUMBER = 10;
    private static final Integer REFERENCE_NUMBER = 4;
``